### ISSUE TITLE
Icon Support for Multi-Select Component

### ISSUE THREAD

**Issue by PrajwalGonsalves07** [primefaces/primeng#2195]
[ ] bug report => Search github for a similar issue or PR before submitting
[X ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35

**Current behavior**
Currently, In the MultiSelect component a text box(label field) is shown as top element.

**Expected behavior**
I want to have a button with an icon instead of a text box.  

**What is the motivation / use case for changing the behavior?**
I am intending to use the MultiSelect component as a ColumnToggler which has multiple columns. The toggler component looks bad when there are lot of checked columns. Instead I want to have a constant icon like the _Watch button_ above in this page. 
It will be great to have this, instead of a label field whose width increases with more columns selected.

* **Angular version:** 2.3.1

* **PrimeNG version:** 2.0.1

* **Browser:** Chrome

**Comment by PrajwalGonsalves07** [comment]
**WorkAround:** As there was no response from the team, we used this workaround:-  We used a button with an icon as a dummy in place of the actual multiselect. On click of the button, we controlled the show and hide of the multiselect component. We hid the multiselect container which contained the label and controlled the body of the multiselect component which displayed all the combo-boxes.
Below is the styling we added in the styles.css 
_.column-toggler .ui-helper-hidden-accessible {
    display: none;
}
.column-toggler .ui-multiselect-label-container {
    display: none;
}
.column-toggler .ui-multiselect-trigger {
    display: none;
}
.column-toggler .ui-state-default .ui-corner-right {
    display: none;
}
.column-toggler .ui-multiselect .ui-widget .ui-state-default ui-corner-all {
    display: none;
}
.column-toggler .ui-multiselect .ui-widget .ui-state-default .ui-corner-all>.ui-multiselect-panel .ui-widget .ui-widget-content .ui-corner-all .ui-shadow {
    display: none;
}_
